### PR TITLE
Added mercurial program to be installed on Ubuntu box via the respective ansible script

### PR DESCRIPTION
**Pull request by neomatrix369**
Was unable to test it using the vagrant/ansible box. The location of the Ubuntu.yml ansible script changed to a new location. 

README.md was amended during the process.

Currently unable to run the command:
```
ansible-playbook -i hosts -s AdoptOpenJDK_Linux_Playbook/roles/Common/tasks/Ubuntu.yml 
``` 
from inside the vagrant box.

There should be a straight-forward way to run this playbook in your local environment.

### CODE REVIEW

**Comment by gdams** [comment]
Why do we need mercurial on the ubuntu boxes?

**Comment by neomatrix369** [comment]
@gdams I would like to run the MercurialTrack program which works only on hg repos, see PR https://github.com/AdoptOpenJDK/openjdk-build/pull/208, hence need `hg` on the Ubuntu box for now

**Comment by neomatrix369** [comment]
@sxa555 Thanks for the heads up, i'll push thru the changes. Can you also please guide me the changes with the placement of the `Ubuntu.yml` script? I couldn't get it to run inside the vagrant box using the usual commands mentioned on the [README.md](https://github.com/AdoptOpenJDK/openjdk-infrastructure/tree/master/ansible), should have worked outside the box despite the change in location.

**Comment by jdekonin** [comment]
This change should be added to  openjdk-infrastructure/ansible/playbooks/AdoptOpenJDK_Linux_Playbook/roles/Common/vars/Ubuntu.yml with the other packages.  @bblondin

**Comment by bblondin** [comment]
Yes this package should be added in ```vars``` not ```tasks```
and under the heading ```Test_Tool_Packages:```

File: ansible/playbooks/AdoptOpenJDK_Linux_Playbook/roles/Common/vars/Ubuntu.yml [hunk 1]
```diff
@@ -61,5 +61,6 @@ Test_Tool_Packages:
   - pulseaudio
   - xauth
   - xorg
+  - mercurial
```
Comment (bblondin) [inline ansible/playbooks/AdoptOpenJDK_Linux_Playbook/roles/Common/vars/Ubuntu.yml]: Could we put this in alphabetical order please?

Comment (neomatrix369) [reply ansible/playbooks/AdoptOpenJDK_Linux_Playbook/roles/Common/vars/Ubuntu.yml]: Done!

Just a query, why is it placed under the `Test_Tool_Packages` category, its a version control system, more like to fall under an appropriate source code / build tools category if we want to be correct about categorisation.

Comment (bblondin) [reply ansible/playbooks/AdoptOpenJDK_Linux_Playbook/roles/Common/vars/Ubuntu.yml]: That's fair. Its up to you if you'd like to move it there.

**Pull request merged**